decidim-core/app/packs/stylesheets/decidim/_header.scss

```
@@ -382,18 +460,61 @@ header {
     }
 
     &__language-chooser {
-      @apply absolute top-full left-0 bg-white rounded w-full bg-gray-5;
+      @apply absolute top-full left-0 rounded w-full bg-gray-5;
+
+      &-desktop {
+        @apply border-r border-gray-6 pr-10;
+
+        & button {
+          @apply border border-gray-7 rounded px-4 py-2 gap-x-2;
+
+          & > span,
+          & > svg:first-of-type {
+            @apply text-gray-8;
+            @apply block #{!important};
```

Same with this and the one on line 483

If it is necessary, then it is better to just use plain CSS and not mix tailwind's apply with important  

```suggestion
            display: block !important;
```